boefjes/tests/integration/test_api.py

```diff
@@ -17,9 +17,9 @@ def test_get_local_plugin(test_client, organisation):
 
 def test_filter_plugins(test_client, organisation):
     response = test_client.get(f"/v1/organisations/{organisation.id}/plugins/")
-    assert len(response.json()) == 101
+    assert len(response.json()) == 102
```

Comment:
Is this test back again? i Thought it was removed at some point due to it being utterly useless